cypress/e2e/ScraperLiteEdition/APIInfo.cy.js

```diff
@@ -0,0 +1,97 @@
+import { SCRAPER_BASE_URL } from '../../resources/data';
+
+describe('Scraper(Lite Edition) APIs Info page', () => {
+    before(() => {
+        cy.visit(SCRAPER_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check for the "API Info" tab is present', () => {
+        cy.componentVisiblityCheck('#details-tab > .d-md-flex > span');
+    });
+
+    it('3. Test to click on the "API Info" tab', () => {
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
+    it('4. Test to click "Adanced Scraper API" link', () => {
+        cy.get('article > :nth-child(4) > a').click();
+        cy.document().should((doc) => {
+            expect(doc.readyState).to.equal('complete');
+        });
+        cy.assertPathname('/marketplace/adv_scraper-api');
+        cy.checkTextVisibility('Advanced Scraper API');
+        cy.go('back');
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
+    it('5. Test to check text "How to use CSS Selectors?" is present', () => {
+        cy.checkTextVisibility('How to use CSS Selectors?');
+    });
+
+    it('6. Test to check text "Image files scraping" is present', () => {
+        cy.checkTextVisibility('Image files scraping');
+    });
+
+    it('7. Test to check text "Rotating IP addresses" is present', () => {
+        cy.checkTextVisibility('Rotating IP addresses');
+    });
+
+    it('8. Test to check text "Setting Custom HTTP Headers" is present', () => {
+        cy.checkTextVisibility('Setting Custom HTTP Headers');
+    });
+
+    it('9. Test to check text "Can I scrape Google Search Results with it?" is present', () => {
+        cy.checkTextVisibility('Can I scrape Google Search Results with it?');
+    });
+
+    it('10. Test to check text "Privacy" is present', () => {
+        cy.checkTextVisibility('Privacy');
+    });
+
+    it('11. Test to check text "Common areas web scraping used for:" is present', () => {
+        cy.checkTextVisibility('Common areas web scraping used for:');
+    });
+
```

Comment:
write test cases to check the ul elements  in the common areas web scraping used for 
![Uploading image.png…]()

Reply:
Added the ul list.